### PR TITLE
fix(component): fixed style SpecialMealCard

### PR DESCRIPTION
**Pull request by Hudayberdiev2000**

### CODE REVIEW

File: src/foundations/sass/_colors.scss [hunk 1]
```diff
@@ -44,3 +44,15 @@ $menu-item-text-color: var(--menu-item-text-color, $secondary-color);
 
 $basket-button-bg: var(--basket-button-bg, $secondary-color);
 $basket-button-text-color: var(--basket-button-text-color, $white-floral);
+
+/* =============================================
+=                 SpecialMealCard              =
+============================================= */
+$special-meal-card-text-color: var(
+  --special-meal-card-text-color,
```
Comment (bruno-keiko) [inline src/foundations/sass/_colors.scss]: Please rename special-meal-card-text-color to special-meal-card-color.

File: src/components/atoms/specialMealCard/SpecialMealCard.stories.tsx [hunk 2]
```diff
@@ -9,11 +10,25 @@ export const Default = {
...
   tags: ["autodocs"],
   excludeStories: /.*Data$/,
+  decorators: [
+    (Story: FC) => (
+      <div
+        style={{
+          backgroundColor: "#fff8ee",
+          height: "180px",
+          width: "180px",
+          borderRadius: "20px",
```
Comment (bruno-keiko) [inline src/components/atoms/specialMealCard/SpecialMealCard.stories.tsx]: Why do you put height, width, borderRadius?

**Review by bruno-keiko — CHANGES_REQUESTED** [review]
We need minor changes below. Please fix the image shifting. When the title becomes two lines, the image shifts but it is not a good experience. Can you fix this problem?

File: src/components/atoms/specialMealCard/SpecialMealCard.stories.tsx (hunk 2 above)
Comment (Hudayberdiev2000) [reply src/components/atoms/specialMealCard/SpecialMealCard.stories.tsx]: I put them to look exactly the same as in figma. I put them just in stories; isn't it just presentation purposes.

**Comment by Hudayberdiev2000** [comment]
> We need minor changes below. Please fix the image shifting. When the title becomes two lines, the image shifts but it is not a good experience. Can you fix this problem?

i will

**Review by bruno-keiko — APPROVED** [review]
Thank you for contributing!

**Pull request merged**